tests/e2e/framework/main_entry.go

```diff
@@ -67,7 +168,7 @@ func (f *Framework) setUp() error {
 // deleting the cluster-wide resources, like roles, service accounts, or the deployment.
 // If we don't properly cleanup resources before deleting CRDs, it leaves resources in a
 // terminating state, making them harder to cleanup.
-func (f *Framework) tearDown() error {
+func (f *Framework) TearDown() error {
 	// Make sure all scans are cleaned up before we delete the CRDs. Scans should be cleaned up
 	// because they're owned by ScanSettingBindings or ScanSuites, which should be cleaned up
 	// by each individual test either directly or through deferred cleanup. If the test fails
```

Comment:
I am wondering if we should clean up in a sequence of ssb, tp, scan, and then scansuite?

Reply:
Through out the series of patches, I added individual clean up to each test so they clean up after themselves.

I don't install other resource except what's required for the tests when I use the e2e target, but if someone did then running the tests might conflict, or cleanup unintended resources (e.g., `deploy-local` and `e2e` on the same cluster?)

Do you use a different workflow?

Reply:
I think that makes sense! thanks for explaining it